README.md

```diff
@@ -77,6 +84,20 @@ cargo clippy
 cargo test
 ```
 
+### VS Code support
```

Comment:
I appreciate the added comments but I think this actually is kind of verbose.
Perhaps splitting it up into bullet points would be more concise.